src/jit/importer.cpp

```diff
@@ -1426,7 +1425,7 @@ GenTree* Compiler::impGetStructAddr(GenTree*             structVal,
         return (structVal->gtObj.Addr());
     }
     else if (oper == GT_CALL || oper == GT_RET_EXPR || oper == GT_OBJ || oper == GT_MKREFANY ||
-             structVal->OperIsSimdHWIntrinsic())
+             structVal->OperIsSIMDorHWintrinsic())
```

Comment:
Just wanted to confirm, this is now also checking for SIMD nodes and is what fixes the bug, the other changes are just simplifying the checks (from two calls into one) and calling the renamed method (`OperIsSimdOrHWIntrinsic()`). Correct?

Reply:
Yes, that's correct.